extensions/wikidata/src/org/openrefine/wikidata/editing/EditBatchProcessor.java

```diff
@@ -67,6 +70,14 @@
     private int globalCursor;
     private Map<String, EntityDocument> currentDocs;
     private int batchSize;
+    private static final String MAX_LAG_KEY = "maxLag";
+    private static final int MAX_LAG_DEFAULT = 5000;      // 5 second default maxLag
+    private PreferenceStore prefStore = ProjectManager.singleton.getPreferenceStore();
+
+    private int getMaxLag() {
+        return prefStore.get(MAX_LAG_KEY) == null ? MAX_LAG_DEFAULT :
+                Integer.parseInt((String) prefStore.get(MAX_LAG_KEY));
```

Comment:
What happens if `parseInt` fails?  Should it fall back to the default?

Reply:
good catch, I should wrap the code in a try catch and use default if user put something invalid.